Makefile

```diff
@@ -42,7 +42,10 @@ GEMINI_FLAGS ?= --fail-fast \
 	--duration=$(DURATION) \
 	--warmup=$(WARMUP) \
 	--profiling-port=6060 \
-	--drop-schema=true
+	--drop-schema=true \
+	--oracle-statement-log-file=$(PWD)/results/oracle-statements.log.zstd \
```

Comment:
Isn't the filename extension should be just '.zst' ?

Reply:
yes, my bad